Updated core/AwardsPlayers.csv to 2021 data

cBrou opened the pull request.

tturocy commented:
Many thanks for proposing this.  However there are some problems:

(1) This changes Appearances.csv, which it should not do.
(2) Please rebase the changes to AwardsPlayers.csv so it is just one commit.
(3) There are conflicts showing in AwardsPlayers.csv.  Ordinarily there should not be, unless there is some good reason for it?  If so please document.

cBrou commented:
Thanks.  

Updated my forked repository to match the current state.  
Cleared up the issue with Appearances.csv.  
Updated AwardsPlayer.csv with values to 2021.  

Not sure what you mean by conflicts showing in AwardsPlayers.csv.

tturocy commented:
Your revisions to AwardsPlayers appears to have resolved the conflicts.

However, the pull request needs to be just one single commit changes only to AwardsPlayers.csv.  You've got a dozen commits in this pull request, including deleting and reinstating files.  This will ruin the change tracking for these files.
